x/mongo/driver/topology/topology_test.go

```diff
@@ -60,6 +60,10 @@ func TestServerSelection(t *testing.T) {
 	var selectError description.ServerSelectorFunc = func(description.Topology, []description.Server) ([]description.Server, error) {
 		return nil, errSelectionError
 	}
+	var errContextError = context.DeadlineExceeded
+	var selectContextError description.ServerSelectorFunc = func(description.Topology, []description.Server) ([]description.Server, error) {
+		return nil, errContextError
```

Comment:
nit: Do we need to declare `errContextError`? Can this be reduced to `return nil, context.DeadlineExceeded`?

Reply:
done